### PR TITLE
tutorial 2 text + fix in visu

### PR DESCRIPTION
**Pull request by jelmerborst**
some small changes to the text of tutorial 2, added a fix to visu which should have been included yesterday but apparently wasn't.

### CODE REVIEW

File: src/hsmm_mvpy/visu.py [hunk 1]
```diff
@@ -174,25 +172,19 @@ def plot_topo_timecourse(channels, estimated, channel_position, init, time_step=
...
         return ax
     else:
         plt.show()    
 
-def plot_components_sensor(hmp_data, positions):
-    from mne.viz import plot_topomap
-    fig, ax = plt.subplots(1,len(hmp_data.attrs['components'].component))
-    for comp in hmp_data.attrs['components'].component:
-        plot_topomap(hmp_data.attrs['components'].values[:,comp], positions, axes=ax[comp], show=False, cmap='Spectral_r')
-    plt.show()
```
Comment (GWeindel) [inline src/hsmm_mvpy/visu.py:194]: Why did you remove this? Is this a version conflict?

File: src/hsmm_mvpy/models.py
Comment (GWeindel) [inline src/hsmm_mvpy/models.py:1]: Our changes will conflict but I can handle that when I push mine

File: src/hsmm_mvpy/visu.py (hunk 1 above)
Comment (jelmerborst) [reply src/hsmm_mvpy/visu.py:194]: this shouldn't have been removed, I'll do a new commit.

**Comment by jelmerborst** [comment]
change made, can be merged

**Review by GWeindel — APPROVED** [review]
